tests/ui/pages/desktop/themes.py

```diff
@@ -0,0 +1,42 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Themes(Base):
+    """ Themes page. """
+
+    _wait_element = (By.CSS_SELECTOR, '.personas-featured > h3')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.find_element(
+            *self._wait_element).is_displayed())
```

Comment:
Looks like you could replace this with `self.featured.themes[0].name` which will be falsy if it's empty or not displayed.